tsdb/chunkenc/xor.go

```diff
@@ -64,6 +64,11 @@ func NewXORChunk() *XORChunk {
 	return &XORChunk{b: bstream{stream: b, count: 0}}
 }
 
+// NewXORChunk returns a new chunk with XOR encoding of the given size.
+func NewXORChunkFromBytes(b []byte) *XORChunk {
```

Comment:
Do we need this? To me it's unnecessary. You have a super quick `chunk.FromData` function. I would drop it and avoid duplicates in API :hugs: